### PR TITLE
feat(django 3): Updates to use with django 3.

### PR DESCRIPTION
**Pull request by Savchenko-Ilia**
## The issue

https://sailo1.atlassian.net/browse/SPM-5373

## The checklist

- [x] I have performed self-review of my own code
- [x] I have tested my code locally
- [x] I have added automatic tests that prove my solution
- [ ] I have made sure the tests cover 85% of the new code
- [ ] I have added type hints to the new code
- [ ] I have made the corresponding changes to the README
- [x] I have assigned one of my peers to review the code
- [x] I have tagged the PR accordingly e.g. bug, feat, wip, ready etc.
- [ ] I have specified the sprint this PR corresponds to

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -72,3 +72,7 @@ deps = django>=2.1,<2.1.99
 [testenv:py3.7-d2.2]
 basepython = python3.7
 deps = django>=2.2,<2.2.99
+
+[testenv:py3.9-d3.2]
+basepython = python3.9
+deps = django>=3.0,<3.2.13
```
Comment (alexei) [inline tox.ini]: To stay in line with the other configs, this should probably say:

```suggestion
deps = django>=3.0,<3.2.99
```

Also don't forget about the new line.

**Review by alexei — CHANGES_REQUESTED** [review]

**Comment by alexei** [comment]
Is it possible to set it up to run tests on GitHub?

**Comment by Savchenko-Ilia** [comment]
> Is it possible to set it up to run tests on GitHub?

will try

**Comment by Savchenko-Ilia** [comment]
> Is it possible to set it up to run tests on GitHub?

Done
https://github.com/sailoboats/django-generic-confirmation/pull/2/commits/48a3a0182a5efc4c2b980bac22e2fcbbb00444ad

**Comment by Savchenko-Ilia** [comment]
@alexei can we merge this ?

**Comment by alexei** [comment]
> @alexei can we merge this ?

I didn't notice you were done, sorry

**Review by alexei — APPROVED** [review]

**Pull request merged**